posthog/constants.py

```diff
@@ -309,6 +309,7 @@ class FlagRequestType(StrEnum):
 BATCH_EXPORTS_TASK_QUEUE = "batch-exports-task-queue"
 SYNC_BATCH_EXPORTS_TASK_QUEUE = "no-sandbox-python-django"
 GENERAL_PURPOSE_TASK_QUEUE = "general-purpose-task-queue"
+USAGE_REPORTS_TASK_QUEUE = "usage-reports-task-queue"
```

Comment:
You can just use the `GENERAL_PURPOSE_TASK_QUEUE` to start off with, then move to your own dedicated pods later on. The task queue basically determines which pod the workflow runs on